### PR TITLE
Abdul/serial uart

### PR DESCRIPTION
**Pull request by v-abdrahman**
Created a folder for SerialUART app & added all the project files for windows 10 Iot Enterprise.

Following changes are done:

1) file modified : SerialSampleCpp.vcxproj
     TargetPlatformVersion from 10.0.17763.0 to 10.0.19041.0 , TargetToolSet from v141 to v142 and TargetPlatformMinVersion from 10.0.10240.0 to 10.0.15063.0.

2) file modified : SerialSampleCs.proj
    TargetPlatformVersion from 10.0.17763.0 to 10.0.19041.0 and TargetPlatformMinVersion from 10.0.10240.0 to 10.0.15063.0.

2) file modified : README.md
    Adding windows 10 IoT Enterprise in products.
    Updated the Windows Platform from IoT Core to IoT Enterprise.
    Changed the Raspberry Pi Board to UpBoard and its Connections.
    Added the app *packages installing steps

### CODE REVIEW

File: SerialUART/CPP/SerialSampleCpp.vcxproj [hunk 1]
```diff
@@ -0,0 +1,194 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup Label="Globals">
+    <ProjectGuid>{b5690b48-a2fa-44f8-a7a4-18af7ea247e6}</ProjectGuid>
+    <RootNamespace>SerialSampleCpp</RootNamespace>
+    <DefaultLanguage>en-US</DefaultLanguage>
+    <MinimumVisualStudioVersion>14.0</MinimumVisualStudioVersion>
+    <AppContainerApplication>true</AppContainerApplication>
+    <ApplicationType>Windows Store</ApplicationType>
+    <WindowsTargetPlatformVersion>10.0.17763.0</WindowsTargetPlatformVersion>
```
Comment (neilsh-msft) [inline SerialUART/CPP/SerialSampleCpp.vcxproj]: This was set to 19041 in other projects, why not this one?

File: SerialUART/CS/README.md [hunk 2]
```diff
@@ -0,0 +1,462 @@
...
+#### <a name="UpBoard_UART"></a>On-board UART (UpBoard)
+
+The UpBoard has an on-board UART. See the [UpBoard pin mapping page](../../Resources/PinMappingsUpBoard.png) for more details on the UpBoard GPIO pins. 
+
+* UART uses GPIO pins 6, 8, 10  
+
+Make the following connections:
+
+* Insert the USB end of the USB-to-TTL cable into a USB port on the PC
+* Connect the GND wire of the USB-to-TTL cable to Pin 6 (GND) on the Upboard
```
Comment (neilsh-msft) [inline SerialUART/CS/README.md:28]: Upboard is spelled Upboard here, but Up-Board or UpBoard elsewhere in this document.  Please make consistent.

File: SerialUART/CS/SerialSample.csproj [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+    <Platform Condition=" '$(Platform)' == '' ">x86</Platform>
+    <ProjectGuid>{82C0DDB8-06E9-49E7-89BB-40B70E2934B1}</ProjectGuid>
+    <OutputType>AppContainerExe</OutputType>
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <RootNamespace>SerialSample</RootNamespace>
+    <AssemblyName>SerialSample</AssemblyName>
+    <DefaultLanguage>en-US</DefaultLanguage>
+    <TargetPlatformIdentifier>UAP</TargetPlatformIdentifier>
+    <TargetPlatformVersion>10.0.19041.0</TargetPlatformVersion>
+    <TargetPlatformMinVersion>10.0.17134.0</TargetPlatformMinVersion>
```
Comment (neilsh-msft) [inline SerialUART/CS/SerialSample.csproj]: This is a different value in other projects is used for TargetPlatformMinVersion.  Why is this one 17134?

Comment (v-abdrahman) [reply SerialUART/CS/SerialSample.csproj]: <TargetPlatformMinVersion>10.0.15063.0</TargetPlatformMinVersion>

I have changed the TargetPlatform Version as per the other Project

File: SerialUART/CS/README.md (hunk 2 above)
Comment (v-abdrahman) [reply SerialUART/CS/README.md:28]: Changed the Up-Board to Upboard in the readme.

File: SerialUART/CPP/SerialSampleCpp.vcxproj (hunk 1 above)
Comment (v-abdrahman) [reply SerialUART/CPP/SerialSampleCpp.vcxproj]: <TargetPlatformVersion>10.0.19041.0</TargetPlatformVersion>

Changed the Target PlatformVersion

File: SerialUART/CS/README.md (hunk 2 above)
Comment (rsameser) [reply SerialUART/CS/README.md:28]: It should be "UP Board" - https://up-board.org/up/specifications/

**Review by rsameser — APPROVED** [review]

File: SerialUART/CS/README.md (hunk 2 above)
Comment (v-abdrahman) [reply SerialUART/CS/README.md:28]: I'm mentioning it as Upboard and keeping it consistent in all the readmes, Hope it is fine?

Comment (neilsh-msft) [reply SerialUART/CS/README.md:28]: Please change it to "UP Board".

**Pull request merged**